reproject/adaptive/high_level.py

```diff
@@ -12,8 +13,9 @@ def reproject_adaptive(input_data, output_projection, shape_out=None, hdu_in=0,
                        order=None,
                        return_footprint=True, center_jacobian=False,
                        roundtrip_coords=True, conserve_flux=False,
-                       kernel='Hann', kernel_width=1.3, sample_region_width=4,
-                       boundary_mode='ignore', boundary_fill_value=0,
+                       kernel='default', kernel_width=1.3,
```

Comment:
The usual way to deal with this is to actually have ``kernel=None`` rather than have ``'default'`` as a special string

Reply:
Ooh, yes, I like that better. Done